lock-impl/src/main/java/com/palantir/lock/logger/LockServiceLoggerTestUtils.java

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright 2017 Palantir Technologies
+ *
+ * Licensed under the BSD-3 License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://opensource.org/licenses/BSD-3-Clause
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.palantir.lock.logger;
+
+import java.io.File;
+import java.io.IOException;
+
+public class LockServiceLoggerTestUtils {
```

Comment:
Tests don't tolerate having unchecked files after tests run. So we need to cleanup after every logging call.